docs/source/llms/llama-index/index.rst

```diff
@@ -58,19 +67,28 @@ use an index.
         </article>
     </section>
 
+
 .. toctree::
     :maxdepth: 2
     :hidden:
 
+    notebooks/llama_index_workflow_tutorial.ipynb
     notebooks/llama_index_quickstart.ipynb
 
 Concepts
 --------
 
+``Workflow`` 🆕
+^^^^^^^^^^^^^^^
```

Comment:
Could we add a "supported as of llamaindex version 'x'" note (and MLflow version 'x' as well :) ) ?